scripts/index.js

```diff
@@ -21,32 +21,36 @@ function handleAdd(e){
                 break;
             }
         }
-            if(m==1){
```

Comment:
The else is for this if statement and handles the condition when the title input is blank
![image](https://user-images.githubusercontent.com/47845897/72835459-c4affd00-3cb0-11ea-835b-1f4fbeed7937.png)
You can't comment it. I think that in the second time out it should be document.getElementById('success').innerHTML=""
instead of  document.getElementById('dupli').innerHTML=""

Reply:
Try uncommenting it, it should work just fine